src/avfoundation.cs

```diff
@@ -9700,6 +9791,17 @@ interface AVCaptureDevice {
 		void RequestAccessForMediaType (AVAuthorizationMediaType mediaType, AVRequestAccessStatus completion);
 #endif
 
+		[iOS (7,0)]
+		[Mac (10,14)]
+		[Static, Export ("requestAccessForMediaType:completionHandler:")]
+		[Async]
+		void RequestAccessForMediaType (NSString avMediaTypeToken, AVRequestAccessStatus completion);
+
+		[iOS (7,0)]
+		[Mac (10,14)]
+		[Static, Export ("authorizationStatusForMediaType:")]
+		AVAuthorizationStatus GetAuthorizationStatus (NSString avMediaTypeToken);
```

Comment:
IIRC there is a strong version of this one inside manual code that you might want to also expose to macOS

Reply:
Are you sure?

iMac:xamarin-macios mandel$ grep GetAuthorizationStatus src/AVFoundation/  -Rn
iMac:xamarin-macios mandel$